compiler/passes/ResolveScope.cpp

```diff
@@ -1001,12 +1001,14 @@ Symbol* ResolveScope::lookupPublicUnqualAccessSyms(const char* name,
   if (!this->canReexport) return NULL;
 
   std::map<Symbol *, astlocT *> renameLocs;
-  Symbol *retval = lookupPublicUnqualAccessSyms(name, context, renameLocs);
+  Symbol *retval = lookupPublicUnqualAccessSyms(name, context, renameLocs,
+                                                true);
   return retval;
 }
 
 Symbol* ResolveScope::lookupPublicUnqualAccessSyms(const char* name,
-         BaseAST *context, std::map<Symbol*, astlocT*>& renameLocs) {
+         BaseAST *context, std::map<Symbol*, astlocT*>& renameLocs,
+                                                   bool followUses) {
```

Comment:
I prefer this arg to be aligned with `BaseAST` in the line above, can ignore if you disagree

Reply:
I agree, it just was a pain because really the BaseAST should be aligned with the const char above it, but the map argument then wraps around.  I'll play with it, I think there's another order that doesn't exceed 80 characters

Reply:
You can typedef that map's type? function name isn't helping either, though I don't want it to be less self-explanatory

Reply:
In the past, I've moved the return type up a line and that usually is sufficient, though it may not be in this case.